src/network.rs

```diff
@@ -22,6 +22,22 @@ pub struct DirectMessageResponse {
     pub timestamp: u64,
 }
 
+/// Node description request/response types
+#[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
```

Comment:
Unused Type Definitions are still present
Either remove these types completely OR refactor to use them instead of string prefixes